internal/cmd/shell.go

```diff
@@ -44,20 +44,20 @@ func DetectShell(target string) Shell {
 	switch target {
 	case "bash":
 		return Bash
-	case "zsh":
-		return Zsh
+	case "elvish":
+		return Elvish
 	case "fish":
 		return Fish
 	case "gzenv":
 		return GzEnv
-	case "vim":
-		return Vim
-	case "tcsh":
-		return Tcsh
 	case "json":
 		return JSON
-	case "elvish":
-		return Elvish
+	case "tcsh":
+		return Tcsh
+	case "vim":
+		return Vim
+	case "zsh":
+		return Zsh
 	}
```

Comment:
I hope this is alright, sorting sortable things is something I do out of habit. It does tend to be nice when looking things up, but can very much drop this commit if you don't wan the noise.